composer.json

```diff
@@ -7,23 +7,46 @@
 		"email": "jason@stallin.gs"
 	}],
 	"require": {
-		"squizlabs/php_codesniffer": "2.8.*",
-		"wimg/php-compatibility": "dev-master",
+		"squizlabs/php_codesniffer": "2.9.*",
+		"wimg/php-compatibility": "^8",
 		"simplyadmire/composer-plugins" : "@dev"
 	},
 	"require-dev": {
-		"wp-coding-standards/wpcs": "dev-master",
+		"wp-coding-standards/wpcs": "0.10",
```

Comment:
Just wondering why... WPCS current version is 0.14.0

Reply:
@felixarntz is looking into this.  I think we had run into some compatibility issues with the dev-master version, but we're going to take another look at it.  The plugin itself doesn't use this, only Travis.

Reply:
Fixed now via ac64111955624166e58e23d72f6ecef8ef96eae1